inc/class-upgrade.php

```diff
@@ -887,7 +887,7 @@ protected function move_indexables_indexation_reason_for_151() {
 	 * else to `false`.
 	 */
 	public function set_indexation_completed_option_for_145() {
-		WPSEO_Options::set( 'indexables_indexation_completed', YoastSEO()->helpers->indexing->get_unindexed_count() === 0 );
+		WPSEO_Options::set( 'indexables_indexation_completed', YoastSEO()->helpers->indexing->get_limited_unindexed_count( 1 ) === 0 );
```

Comment:
Why not the `get_limited_filtered_unindexed_count` here, like in the indexing notification integration?